src/api/java/com/minecolonies/api/entity/ai/statemachine/states/AIWorkerState.java

```diff
@@ -163,6 +163,30 @@
      */
     FARMER_HARVEST(false),
 
+      /*
+###Gravedigger###
+    */
+
+    /**
+     * Empty The grave
+     */
+    EMPTY_GRAVE(false),
+
+    /**
+     * Dig The grave
+     */
+    DIG_GRAVE(false),
+
+    /**
+     * Burry the citizen
+     */
+    BURRY_CITIZEN(false),
```

Comment:
```suggestion
    BURY_CITIZEN(false),
```
For the other instances too ofc